Mathlib/GroupTheory/GroupAction/Primitive.lean

```diff
@@ -50,6 +49,21 @@ import Mathlib.GroupTheory.GroupAction.Transitive
 - `MulAction.IsPreprimitive.isQuasipreprimitive`
   Preprimitive actions are quasipreprimitive.
 
+## Particular results for actions on finite types
+
+- `isPreprimitive_of_primeCard` :
```

Comment:
Do this results exist somewhere? I don't see them in current mathlib and you are modifying the docstring of this file.

Reply:
Sorry, this belongs to #12053, it had been included at some point but I deleted it from this PR to which it is not necessary.